### PR TITLE
doc: update members after membership review

### PR DESCRIPTION
**Pull request by mhdawson**
Update membership as per
https://github.com/nodejs/diagnostics/issues/217.

### CODE REVIEW

**Comment by mhdawson** [comment]
Will update team as well once this is approved.

File: README.md [hunk 1]
```diff
@@ -39,22 +36,20 @@ Work needed includes:
...
 - Monthly Meetings
 - Biannual F2F
 
 ### Members
 
 <!-- ncu-team-sync.team(nodejs/diagnostics) -->
 
+Current Members#
+
+<!-- ncu-team-sync.team(nodejs/diagnostics) -->
```
Comment (watson) [inline README.md]: Is it on purpose to have two of these comments next to each other?

**Comment by mhdawson** [comment]
@watson good catch I had duplicates of the ncu header. Fixed now.

File: README.md [hunk 2]
```diff
@@ -39,22 +36,18 @@ Work needed includes:
 | Platform neutrality   |               |                                                  |
 
 ### Logistics
-
 - Monthly Meetings
 - Biannual F2F
 
 ### Members
 
+Current Members#
```
Comment (BridgeAR) [inline README.md]: Is the `#` intentional?

File: README.md [hunk 3]
```diff
@@ -39,22 +36,18 @@ Work needed includes:
...
+
 <!-- ncu-team-sync.team(nodejs/diagnostics) -->
 
 - [@AndreasMadsen](https://github.com/AndreasMadsen) - Andreas Madsen
-- [@bnoordhuis](https://github.com/bnoordhuis) - Ben Noordhuis
-- [@BridgeAR](https://github.com/BridgeAR) - Ruben Bridgewater
```
Comment (BridgeAR) [inline README.md]: Hm, I was added recently. Was it a mistake that I was added to this list?

File: README.md (hunk 2 above)
Comment (mhdawson) [reply README.md]: No

File: README.md (hunk 3 above)
Comment (mhdawson) [reply README.md]: Yes, my mistake, you were added after the review started and I messed up. Will fix.

**Comment by mhdawson** [comment]
@BridgeAR fixed.

**Pull request merged**